priv/repo/seeds.exs

```diff
@@ -9,3 +9,14 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+
+alias CodestoriesReviewBot.Repo
+alias CodestoriesReviewBot.Reviews.Category
+
+Repo.insert! %Category{name: "Android"}
```

Comment:
I think it would be better if we stored `name` downcased and then format input from command to make it case-insensitive. WDYT?

Reply:
I think we should downcase it when we read it from DB to avoid the cases when we have to remember about proper format when new category is added.

Reply:
The thing is we're getting Category by `name`, so when user types `join category RoR` he has to remember the correct formatting. But we don't have to store names downcased. We can also do the DB lookup by name like that:
```
  def get_category_by_name(name) do
    downcased_name = String.downcase(name)
    (from c in Category, where: fragment("lower(?)", c.name) == ^downcased_name)
    |> Repo.one()
  end
```